python_modules/libraries/dagster-dbt/dagster_dbt/asset_utils.py

```diff
@@ -113,8 +114,15 @@ def default_freshness_policy_fn(node_info: Mapping[str, Any]) -> Optional[Freshn
     if freshness_policy:
         return freshness_policy
 
-    # TODO: Remove this legacy configuration in 0.20.0
     legacy_freshness_policy_config = node_info["config"].get("dagster_freshness_policy", {})
+    deprecation_warning(
```

Comment:
doesn't this end up emitting warnings for people who have no freshness policy set at all? I think we should do this like:

```python
legacy_freshness_policy = _legacy_freshness_policy_fn(legacy_freshness_policy_config)
if legacy_freshness_policy:
    # deprecation warning
return legacy_freshness_policy
```